.eslintrc.json

```diff
@@ -31,8 +31,15 @@
         "eslint-config-prettier/@typescript-eslint" // Disable typescript-eslint rules that conflict with Prettier
       ],
       "rules": {
-        "@typescript-eslint/explicit-function-return-type": "off", // inference is usually useful
+        "@typescript-eslint/explicit-module-boundary-types": "off", // inference is usually useful, even for module boundaries
         "@typescript-eslint/no-unused-vars": "off", // TypeScript itself checks this via the "noUnusedLocals"/"noUnusedParameters" options
+        "@typescript-eslint/ban-ts-comment": [
```

Comment:
Maybe some comments explaining why these rules are being applied?

Reply:
Is this not descriptive enough? https://github.com/cloudfour/cloudfour.com-patterns/pull/728/files#diff-df39304d828831c44a2b9f38cd45289cR39

Reply:
Ah, sorry, I missed that one. 

Could you add a comment on why we're adding prefer optional chain?

Reply:
✅